src/Contracts/ExplorerContracts.ts

```diff
@@ -30,6 +30,7 @@ export enum MessageTypes {
   ExplorerClickEvent,
   LoadingStatus,
   GetArcadiaToken,
+  GetAadToken,
```

Comment:
Doesn't the order matter here? This gets transpile to a number. I think it has to go at the end

Reply:
the types don't fail but I can remove this either way. I don't need it yet